yarn.lock

```diff
@@ -3039,6 +3039,11 @@ map-visit@^1.0.0:
   dependencies:
     object-visit "^1.0.0"
 
+material-design-icons-iconfont@^5.0.1:
```

Comment:
I presume this thing would pull in the font, right? Where would it activate the font?

Reply:
It pulls in the font, then gets imported in the bundle like hls.js and other external modules.

The current Webpack process will copy it to the dist directory.